docs/1.0. Certificate Provisioning.md

```diff
@@ -67,47 +67,48 @@ The entity that is using the NMOS API, for example:
-2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using Unicast DNS-SD.
-3. The EST Client should then request the Root CA for the target network, from the EST Server
-    - The EST Client should assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
-    - Using the Root CA returned to secure further communications with the EST Server and NMOS Servers
-4. The EST Client must generate a Certificate Signing Request(CSR) for all supported digital signature algorithms with appropriate key lengths
-    - Included in the CSR must be the DNS resolvable name for the current domain of the EST Client
-    - Appropriate values for the remaining CSR fields should be used
-5. The EST Client should send each CSR to the EST Server
-    - The EST Client must provide the manufacturer issued certificate during the TLS handshake, if valid
-    - The EST Server will use the TLS Client Certificate to authenticate and then authorise whether the EST Client. If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate.
-6. The EST Client should provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs
+1. Before the NMOS Server (EST Client) is shipped from the factory it must be provisioned with a unique TLS Client Certificate, signed by the manufacturer's Certificate Authority.
+2. When the EST Client is connected to the target environment's network, it will first discover the location of the EST Server using unicast DNS-SD.
+3. The EST Client will then request the Root CA for the target network, from the EST Server.
+   - The EST Client can assume the EST server found using DNS-SD is trusted and need not perform TLS authentication of the EST Server for this initial transaction.
+   - It uses the Root CA returned to secure further communications with the EST Server and (?) NMOS Servers.
+4. The EST Client will generate a Certificate Signing Request (CSR) for all supported digital signature algorithms with appropriate key lengths.
+   - Ensure the DNS resolvable name for the current domain of the EST Client is included in the CSR.
+   - Use appropriate values for the remaining CSR fields.
+5. The EST Client will send each CSR to the EST Server.
+   - The EST Client will provide the manufacturer-issued certificate during the TLS handshake, if this certificate is valid.
+   - The EST Server will use the TLS Client Certificate to authenticate and then authorize the EST Client. If the EST Client meets the local CA's security policy, it will be issued with a TLS Server Certificate.
+6. The EST Client will provide the returned TLS Certificate and the corresponding chain of trust in all future responses returned by its NMOS APIs.
```

Comment:
```suggestion
4. The NMOS Server will generate a Certificate Signing Request (CSR) for all supported digital signature algorithms with appropriate key lengths.
   - Ensure the DNS resolvable name for the current domain of the NMOS Server is included in the CSR.
   - Use appropriate values for the remaining CSR fields.
5. The NMOS Server will send each CSR to the EST Server.
   - The NMOS Server will provide the manufacturer-issued certificate during the TLS handshake, if this certificate is valid.
   - The EST Server will use the TLS Client Certificate to authenticate and then authorize the NMOS Server. If the NMOS Server meets the local CA's security policy, it will be issued with a TLS Server Certificate.
6. The NMOS Server will provide the returned TLS Certificate and the corresponding chain of trust in all future responses from its NMOS APIs.
```